### PR TITLE
🌱 Made API registration work for plain as well as controller-runtime

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR updates the way that the API types are registered with one or more `Scheme` objects so that it works with plain kube code as well as controller-runtime based code.

## Related issue(s)

Fixes #

### CODE REVIEW

File: api/control/v1alpha1/register.go [hunk 1]
```diff
@@ -17,6 +17,8 @@ limitations under the License.
 package v1alpha1
 
 import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
```
Comment (nirrozenbaum) [inline api/control/v1alpha1/register.go:21]: why do we need `k8sruntime` alias?

File: api/control/v1alpha1/groupversion_info.go [hunk 2]
```diff
@@ -21,17 +21,16 @@ limitations under the License.
 package v1alpha1
 
 import (
+	k8sruntime "k8s.io/apimachinery/pkg/runtime"
```
Comment (nirrozenbaum) [inline api/control/v1alpha1/groupversion_info.go:24]: same question.. you can use just `runtime.NewSchemeBuilder`

File: api/control/v1alpha1/register.go (hunk 1 above)
Comment (MikeSpreitzer) [reply api/control/v1alpha1/register.go:21]: It's not the only "runtime" in our world.

**Comment by MikeSpreitzer** [comment]
The force-push to 490bb4514 makes the changes that @nirrozenbaum suggested above.

**Review by nirrozenbaum — APPROVED** [review]
/lgtm
Thanks!

**Pull request merged**